lib/fog/compute/kubevirt/models/vm_base.rb

```diff
@@ -43,7 +43,7 @@ def parse_object(object)
             :volumes          => parse_volumes(spec[:volumes]),
             :interfaces       => parse_interfaces(domain[:devices][:interfaces]),
             :networks         => parse_networks(spec[:networks]),
-            :state            => object[:spec][:running].to_s == "true" ? "running" : "stopped",
+            :status           => object[:spec][:running].to_s == "true" ? "running" : "stopped",
```

Comment:
This change will cause https://github.com/fog/fog-kubevirt/blob/master/lib/fog/kubevirt/compute/models/server.rb#L23 not to work:
```
        def ready?
          running?(status) && running?(state) && !ip_address.empty?
        end
```

But perhaps a better option would be removing this method from this file and adding it to foreman_kubevirt https://github.com/masayag/foreman_kubevirt/blob/master/app/models/concerns/fog_extensions/kubevirt/server.rb

Reply:
we can fix it here. I will push an update

Reply:
It should work now. I fixed it in server model. We were defining one property in one model and assign value in the other. Now we define and populate in the same model.